source/BaselineOfStargate/BaselineOfStargate.class.st

```diff
@@ -21,114 +22,139 @@ BaselineOfStargate >> baseline: spec [
 
 	^ MetacelloCypressBaselineProject
 ]
 
-{ #category : #baselines }
+{ #category : 'baselines' }
+BaselineOfStargate >> setUpCorePackages: spec [
+
+	spec
+		package: 'Stargate-Model' with: [
+			spec requires:
+					#( 'Buoy-Deployment' 'Hyperspace-Deployment'
+					   'Teapot-Deployment' 'JSONWebToken-Deployment' ) ];
+		group: 'Core' with: 'Stargate-Model';
+		group: 'Deployment' with: 'Stargate-Model'.
+
+	#( 'Stargate-Zinc-Extensions' 'Stargate-Teapot-Extensions'
+	   'Stargate-NeoJSON-Extensions' 'Stargate-JSONWebToken-Extensions'
+	   'Stargate-Hyperspace-Extensions' 'Stargate-Extensions' ) do: [
+		:extensionPackageName |
+		spec
+			package: extensionPackageName
+			with: [ spec requires: 'Stargate-Model' ];
+			group: 'Core' with: extensionPackageName;
+			group: 'Deployment' with: extensionPackageName ]
+]
+
+{ #category : 'baselines' }
 BaselineOfStargate >> setUpDependencies: spec [
 
 	spec
-		baseline: 'Buoy' with: [ spec repository: 'github://ba-st/Buoy:v6' ];
+		baseline: 'Buoy' with: [ spec repository: 'github://ba-st/Buoy:v7' ];
 		project: 'Buoy-Deployment' copyFrom: 'Buoy' with: [ spec loads: 'Deployment' ];
 		project: 'Buoy-SUnit' copyFrom: 'Buoy' with: [ spec loads: 'Dependent-SUnit-Extensions' ].
 
 	spec
-		baseline: 'Hyperspace' with: [ spec repository: 'github://ba-st/Hyperspace:v4' ];
+		baseline: 'Hyperspace' with: [ spec repository: 'github://ba-st/Hyperspace:v5' ];
 		project: 'Hyperspace-Deployment' copyFrom: 'Hyperspace' with: [ spec loads: 'Deployment' ];
     project: 'Hyperspace-Tools' copyFrom: 'Hyperspace' with: [ spec loads: 'Tools' ];
 		project: 'Hyperspace-SUnit'
 		copyFrom: 'Hyperspace'
 		with: [ spec loads: 'Dependent-SUnit-Extensions' ].
 
 	spec
-		baseline: 'Teapot' with: [ spec repository: 'github://zeroflag/Teapot:v2.7.0/source' ];
+		baseline: 'Teapot' with: [ spec repository: 'github://ba-st-dependencies/Teapot:v2' ];
 		project: 'Teapot-Deployment' copyFrom: 'Teapot' with: [ spec loads: 'Deployment' ].
 
 	spec
-		baseline: 'JSONWebToken' with: [ spec repository: 'github://noha/JSONWebToken:0.3/source' ];
+		baseline: 'JSONWebToken' with: [ spec repository: 'github://ba-st-dependencies/JSONWebToken:v1' ];
 		project: 'JSONWebToken-Deployment'
 		copyFrom: 'JSONWebToken'
-		with: [ spec loads: 'JSONWebToken-Core' ].
+		with: [ spec loads: 'SSL' ].
 
 	spec
-		baseline: 'JRPC' with: [ spec repository: 'github://juliendelplanque/JRPC:v3.x.x/src' ];
+		baseline: 'JRPC' with: [ spec repository: 'github://ba-st-dependencies/JRPC:v4' ];
 		project: 'JRPC-Deployment' copyFrom: 'JRPC' with: [ spec loads: 'Server-Deployment' ].
 
 	spec
-		baseline: 'Launchpad' with: [ spec repository: 'github://ba-st/Launchpad:v4' ];
+		baseline: 'Launchpad' with: [ spec repository: 'github://ba-st/Launchpad:v5' ];
 		project: 'Launchpad-Deployment' copyFrom: 'Launchpad' with: [ spec loads: 'Deployment' ].
 
 	spec
 		baseline: 'ZTimestamp' with: [ spec repository: 'github://svenvc/ztimestamp' ];
```

Comment:
Does the dependecy with ZTimestamp still exist? Now that stack/fuel dump file name generation includes hhmmss + uuid I think we could avoid loading even in Pharo. Or can be removed without too much effort maybe?